debian/changelog

```diff
@@ -1,16 +1,4 @@
-metee (4.1.0-0ubuntu1~ppa3) oracular; urgency=medium
-
-  * Fix dependencies.
-
- -- Pedro Kopper <pedro.kopper@canonical.com>  Wed, 14 Aug 2024 16:10:46 -0300
-
-metee (4.1.0-0ubuntu1~ppa2) oracular; urgency=medium
-
-  * Cleanup comments and extra directories.
-
- -- Pedro Kopper <pedro.kopper@canonical.com>  Wed, 14 Aug 2024 15:50:43 -0300
-
-metee (4.1.0-0ubuntu1~ppa1) oracular; urgency=medium
+metee (4.1.0-0ubuntu1~24.10~ppa1) oracular; urgency=medium
```

Comment:
@mckees  Since you are squashing the changelog, can you also remove `Cleanup comments and extra directories` from the changelog and also set you as author with the right date (right now Pedro is the author with the old date)

Reply:
Alright, I've updated that now